clang/test/CodeGen/cheri/riscv/riscv-default-ir-features.c

```diff
@@ -0,0 +1,61 @@
+// RUN: %clang --target=riscv32-elf -S -emit-llvm %s -o - \
```

Comment:
Don't need -elf

Reply:
I originally also checked the `target triple` in the IR and there was some weird thing going on where only some of the RUN: lines canonicalized it. But I see I removed that CHECK line so I'll drop the -elf.